src/components/facets/facet-container.component.html

```diff
@@ -23,13 +23,14 @@
     </div>
 
     <!-- Display Tags For Selected Items -->
-    <div class="facets-selected-list">
+    <div class="facets-selected-list" uxReorderable [(reorderableModel)]="facets">
```

Comment:
Thanks very much for the PR, works great! I have a few very minor suggestions:

1. We try to retain existing default functionality when adding new features such as this, so by default facet items should remain fixed and reordering should be enabled using an input. This way anyone already using this component won't automatically get reorderable facets if they don't want them.

Now currently our Reorderable directive doesn't have the option to disable reordering, but this should be very easy so add:

**reorderable.directive.ts**
```typescript
@Input() reorderingDisabled: boolean = false;
```
and add some logic to the canMove function

```typescript
canMove(element: Element, container: Element, handle: Element): boolean {
    if (this.reorderingDisabled) {
        return false;
    }

    return this.handles.length === 0 ? true : !!this.handles.find(_handle => _handle.nativeElement === handle);
}
```

Now we can add an input to the facet-container component to enabled reordering:

**facet-container.component.ts**

```typescript
@Input() facetsReorderable: boolean = false;
```

And now only allow reordering when facetReordering is enabled

**facet-container.component.html**

```html
<div class="facets-selected-list" uxReorderable [reorderingDisabled]="!facetsReorderable [(reorderableModel)]="facets>
```


2. We should emit the facetsChange event when reordering occurs so the the application can perform events on reorder. This can be done by adding the following `(reorderableModelChange)="facetsChange.emit(facets)"` to the `.facets-selected-list` element, eg:

```html
<div class="facets-selected-list" uxReorderable [reorderingDisabled]="!facetsReorderable" [(reorderableModel)]="facets" (reorderableModelChange)="facetsChange.emit(facets)">
```

3. This is just a slight appearance change. It could be helpful to slightly alter the display of a tag that is being dragged eg:

![image](https://user-images.githubusercontent.com/20795331/41287013-48160f9c-6e39-11e8-91a1-80bcbd441c15.png)

The reorderable directive adds a `.ux-reorderable-moving` class to any element being reordered so we should just be able to add some styling, for example, reducing the opacity:

**facet-container.component.less**

```less
.facet-selected-tag {
    margin-right: 5px;
    margin-bottom: 5px;
    cursor: pointer;
    outline: none;
    max-width: 100%;
    color: @tag-input-tag-color;
    background-color: @tag-input-tag-bg;
    border: 1px solid @tag-input-tag-border;
    border-radius: 1px;
    padding: 0 5px;

    &.ux-reorderable-moving {
         opacity: 0.5;
    }
```

4. With these changes we have added a new input to both the reorderable directive and our facet container component. It would be great if the documentation was updated just to include these two inputs.

Thanks very much for your contribution, if you have any questions or issues let me know!